pkg/api/frontendsettings.go

```diff
@@ -173,6 +173,7 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 			"sort":          getPanelSort(panel.Id),
 			"skipDataQuery": panel.SkipDataQuery,
 			"state":         panel.State,
```

Comment:
Needed for the frontend changes?

Reply:
Exactly. In order to render labels for panels in the visualizations view

Reply:
Understood